isis/src/control/objs/BundleSettings/BundleSettings.cpp

```diff
@@ -450,14 +450,17 @@ namespace Isis {
   BundleObservationSolveSettings
       BundleSettings::observationSolveSettings(QString observationNumber) const {
 
+    BundleObservationSolveSettings defaultSolveSettings;
```

Comment:
Do we want to return a default here? I think that it would make more sense to return NULL when a BundleSettings object doesn't have a BOSS corresponding to the given observation number.